app/src/main/java/com/example/spacechatapp/presentation/ui/bottomFragment/BottomFragment.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.spacechatapp
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.spacechatapp.databinding.FragmentBottomBinding
+import com.example.spacechatapp.presentation.ui.baseFragment.BaseFragment
+import com.example.spacechatapp.presentation.ui.baseFragment.Inflater
+
+
+class BottomFragment : BaseFragment<FragmentBottomBinding>() {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
```

Comment:
you already have onViewCreated in BaseFragment, create abstract method in base for example onBind() and call it from baseFragment - onViewCreated

Reply:
also move inflate function above onViewCreated()

Reply:
done